lib/Runtime/Library/ScriptFunction.cpp

```diff
@@ -457,16 +457,6 @@ namespace Js
 
         ScriptContext * scriptContext = this->GetScriptContext();
 
-        if (isDefaultConstructor)
```

Comment:
Changed in both locations and removed isDefaultConstructor/DefaultConstructor flag throughout.